lua/pac3/core/client/parts/lock.lua

```diff
@@ -0,0 +1,523 @@
+include("pac3/extra/shared/net_combat.lua")
```

Comment:
Why do you include this here? same goes for other places where you add include. It will rerun all the net_combat code for every inclusion, and it might cause issues with local state if net_combat.lua or any other file has it.

It should be enough to include it once in init before parts are loaded.

Reply:
I don't actually remember that one. Why indeed?